### ISSUE TITLE
set-env deprecation warning

### ISSUE THREAD

**Issue by bcomnes** [olegtarasov/get-tag#14]
Looks like this action is triggering the following warning:

```
Warning: The `set-env` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

Maybe just a matter of updating deps?

**Comment by ahmetkaftan** [comment]
Now it's an error: 
```
Error: The `set-env` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

```

**Comment by bcomnes** [comment]
This was blocking my deploys yesterday so I decided to fork.  

https://github.com/little-core-labs/get-git-tag

**Comment by olegtarasov** [comment]
Hey guys, sorry it took so much time for me to get to this issue. The problem is now fixed, just use `olegtarasov/get-tag@v2.1`.
